packages/shared/src/components/search/SearchPanel/SearchPanelSourceSuggestions.tsx

```diff
@@ -38,7 +38,12 @@ const PanelItem = ({ suggestion, ...rest }: PanelItemProps) => {
   });
 
   return (
-    <SearchPanelItem icon={<Icon />} {...itemProps} {...rest}>
+    <SearchPanelItem
+      icon={<Icon />}
+      {...itemProps}
+      {...rest}
+      className="px-2 py-1"
```

Comment:
Noticed this was wrong compared to design and with the longer list it made it too big

Reply:
Only maybe merge with `rest.className` with `classNames`